plugins/terraform-backend/src/schema/openapi.generated.ts

```diff
@@ -3,136 +3,135 @@
 // ******************************************************************
 // * THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY. *
 // ******************************************************************
-import { createValidatedOpenApiRouter } from '@backstage/backend-openapi-utils';
+import {createValidatedOpenApiRouter} from '@backstage/backend-openapi-utils';
```

Comment:
boo

Reply:
this is ignored by prettier as it's a generated file